### PR TITLE
Add cors config for production

### PR DESCRIPTION
**Pull request by lamuria**
Add cors config for production and add a proper env port name

### CODE REVIEW

File: src/weareswat/meowallet_integration/http_component.clj [hunk 1]
```diff
@@ -58,7 +59,7 @@
 
 (defn get-port
   []
-  (Integer/parseInt (or (env :port) "5050")))
+  (Integer/parseInt (or (env :meowallet-integration-port) "5050")))
```
Comment (donbonifacio) [inline src/weareswat/meowallet_integration/http_component.clj]: You also need the port one for heroku!

**Comment by paulosilva86** [comment]
🆗

File: src/weareswat/meowallet_integration/http_component.clj (hunk 1 above)
Comment (lamuria) [inline src/weareswat/meowallet_integration/http_component.clj]: done

**Comment by donbonifacio** [comment]
🆗

**Pull request merged**